### PR TITLE
Format initial photo display

### PR DESCRIPTION
**Pull request by sbtorres**
@elimoreh can you please review?

This PR formats the initial five photos that are displayed on the Airbnb photo gallery when a listing is opened. This PR also includes functionality for mouse over events. When hovered over, the image will scale 1.05x and the non-hovered images will reduce their opacity to 0.7. 

Thanks!

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
+    this.state = { photos: [], isContainerHovered: false };
 
-    this.state = { photos: [] };
+    this.onContainerMouseOver = this.onContainerMouseOver.bind(this);
+    this.onContainerMouseOut = this.onContainerMouseOut.bind(this);
   }
 
   componentDidMount() {
-    axios.get('/photos/?listingid=2')
+    axios.get('/photos/?listingid=7')
```
Comment (elimoreh) [inline client/src/App.jsx]: Good to hardcode for now, but eventually you'll have to make it take id from the url you type in the browser, the code to get the id number is: window.location.href.match(/listingid\s*=\s*(.*)/)

File: client/src/App.jsx [hunk 2]
```diff
@@ -19,12 +32,33 @@ class App extends React.Component {
...
+  }
+
   render() {
-    const { photos } = this.state;
+    const { photos, index, isContainerHovered } = this.state;
     return (
-      <div>
-        <PhotosContainer photos={photos} />
-      </div>
+      <StyledContainer
```
Comment (elimoreh) [inline client/src/App.jsx]: I cant say much on the style stuff, but to my eye it looks really good!

File: client/src/Photo.jsx [hunk 3]
```diff
@@ -1,12 +1,99 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
 
-const Photo = ({ photo }) => (
-  <img src={photo.photo_url} alt="home interior" />
-);
+const StyledPhoto = styled.div`
+  grid-column-start: ${(props) => {
```
Comment (elimoreh) [inline client/src/Photo.jsx]: Do you need to deconstruct index for styling?

File: client/src/App.jsx (hunk 1 above)
Comment (sbtorres) [reply client/src/App.jsx]: Yeah, it's on my list of things to complete tomorrow. Thanks!

File: client/src/Photo.jsx (hunk 3 above)
Comment (sbtorres) [reply client/src/Photo.jsx]: I did try to do it with object destructuring, but it was not working. My understanding is that you do not when using styled-components. I'll add a task to my trello board to look into this more. Thanks!

**Pull request merged**